docs/src/pages/demos/transfer-list/TransferList.js

```diff
@@ -0,0 +1,175 @@
+import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import Grid from '@material-ui/core/Grid';
+import List from '@material-ui/core/List';
+import ListItem from '@material-ui/core/ListItem';
+import ListItemText from '@material-ui/core/ListItemText';
+import Checkbox from '@material-ui/core/Checkbox';
+import Button from '@material-ui/core/Button';
+
+const useStyles = makeStyles(theme => ({
+  root: {
+    margin: 'auto',
+  },
+  list: {
+    minWidth: 180,
+    height: 230,
+    backgroundColor: theme.palette.background.paper,
+    overflow: 'scroll',
+  },
+  listItem: {
+    paddingTop: 4,
+    paddingBottom: 4,
+  },
+  button: {
+    margin: `${theme.spacing(0.5)}px 0px`,
```

Comment:
```suggestion
    margin: theme.spacing(0.5, 0),
```

Reply:
Ah yes, nice feature! I had forgotten it was included.